pkg/sql/schemachanger/scpb/scpb.proto

```diff
@@ -102,7 +102,7 @@ message TargetState {
 
 message Statement {
   string statement = 1;
```

Comment:
I don't fully understand the purpose of this message but is it worth just keeping `redacted_statement` and forcing callers to call `StripMarkers()` when they need to?

Reply:
that seems viable. I'd prefer to do that separate from this PR, since the `statement` is used within the schema changer for functional behavior, while the `redacted_statement` is just used for logs or EXPLAIN output.